packages/core/src/vivliostyle/counters.ts

```diff
@@ -166,8 +166,12 @@ class CounterResolver implements CssCascade.CounterResolver {
       `page-counter-${name}`,
     );
 
-    const arrayFormat = (arr) => {
-      return format(arr[0]);
+    const arrayFormat = (arr: number[]) => {
+      let value = arr[0];
+      if (arr[1]) {
+        value = arr[1];
+      }
+      return format(value);
```

Comment:
```suggestion
      return format(arr[arr.length - 1]);
```

`value = arr[1]` is not good when `counter-reset: page` happens multiple times, for example: 

      @page :nth(2) {
        counter-reset: page 20;
      }
      @page :nth(3) {
        counter-reset: page 30;
      }